src/main/java/org/veupathdb/service/eda/compute/plugins/correlation/CorrelationPlugin.java

```diff
@@ -167,10 +167,7 @@ protected void execute() {
     // Get data stream(s) for Rserve
     HashMap<String, InputStream> dataStream = new HashMap<>();
     dataStream.put(INPUT_DATA, getWorkspace().openStream(INPUT_DATA));
-    boolean hasSecondCollection = computeConfig.getCollectionVariable2() != null && !computeConfig.getCollectionVariable2().getCollectionId().equals(METADATA_COLLECTION_ID);
```

Comment:
why remove rather than modify this logic? presumably hasSecondCollection is easier to read and still accurate. also why are we not adding the second stream any longer?